packages/cli/src/command-utils/common-options.ts

```diff
@@ -17,65 +22,67 @@ export const OPTIONS = {
     string: true,
   },
   headless: {
+    // Note: when running in CI we default to headless (see DEFAULT_EXECUTION_OPTIONS), but for local runs via the CLI we use headed mode by default
     boolean: true,
     description: "Start browser in headless mode",
     default: false,
   },
   devTools: {
     boolean: true,
     description: "Open Chrome Dev Tools",
-    default: false,
+    default: DEFAULT_EXECUTION_OPTIONS.devTools,
   },
   bypassCSP: {
     boolean: true,
     description:
       "Enables bypass CSP in the browser (danger: this could mean you tests hit your production backend)",
-    default: false,
+    default: DEFAULT_EXECUTION_OPTIONS.bypassCSP,
   },
   shiftTime: {
     boolean: true,
     description: "Shift time during simulation to be set as the recording time",
-    default: true,
+    default: DEFAULT_EXECUTION_OPTIONS.shiftTime,
   },
   networkStubbing: {
     boolean: true,
     description: "Stub network requests during replay",
-    default: true,
+    default: DEFAULT_EXECUTION_OPTIONS.networkStubbing,
   },
   skipPauses: {
     boolean: true,
     description:
       "Fast forward through any pauses to replay as fast as possible.",
-    default: true,
+    default: DEFAULT_EXECUTION_OPTIONS.skipPauses,
   },
   moveBeforeClick: {
     boolean: true,
     description: "Simulate mouse movement before clicking",
-    default: true,
+    default: DEFAULT_EXECUTION_OPTIONS.moveBeforeClick,
   },
   diffThreshold: {
     number: true,
     description:
       "Acceptable maximum proportion of changed pixels, between 0 and 1. If this proportion is exceeded then the test will fail.",
-    default: 0.01,
+    default: DEFAULT_SCREENSHOTTING_OPTIONS.diffOptions.diffThreshold,
```

Comment:
This changes it from 0.01 (10k pixels on a 1000 by 1000px screenshot, or 100 pixels by 100 pixels) to 0.00001 (10 pixels on a 1000 by 1000px screenshot). But I think it makes sense for the CLI defaults to match the CI defaults, otherwise debugging can be misleading. Also 100 pixels by 100 pixels was _**far**_ too large